webrtc.html

```diff
@@ -1816,9 +1795,20 @@
                         </ol>
                       </li>
                       <li>
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        equal to any ufrag present in the corresponding
+                        <a>media description</a> of an applied remote
+                        description, reject <var>p</var> with an
```

Comment:
Hmm. It could definitely be either the pending/current description (rollback is the main reason we need to differentiate with the ufrag in the first place). But could it be anything more than the pending/current description? Is it expected that after applying an answer and the ICE restart is accepted, the old ICE generation immediately aborts? If so we should specify that in JSEP.